modules/EsetTools.py

```
@@ -290,14 +315,85 @@ def getLicenseData(self):
             console_log('Information successfully received!', OK)
-            license_name = 'ESET Endpoint Security + ESET Server Security'
             return license_name, license_key, license_out_date
+
+def EsetVPNReset(key_path='SOFTWARE\\ESET\\ESET VPN', value_name='authHash'):
+    """Deletes the authHash value of ESET VPN"""
+    if not sys.platform.startswith('win'):
+        raise RuntimeError('This feature is for Windows only!!!')
+    try:
+        check_output(['taskkill', '/f', '/im', 'esetvpn.exe'], stderr=DEVNULL).decode('cp866', errors="ignore")
+    except:
+        pass
```

Ruff (0.8.0)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)